src/components/ListView.jsx

```diff
@@ -50,21 +48,54 @@ function ListView() {
   //
   const handleCheckboxChange = async (e) => {
     const itemId = e.target.name;
-    //create time variable to save time when user checked the box as purchasedTime
-    const datePurchased = Date.now();
-    //if user want to uncheck the item it can be done and purchasedDate is set to null again
+    // grabbing the specific item from state that is clicked because we will be updating its properties
+    const itemToUpdate = items.find((item) => itemId === item.itemName);
```

Comment:
We can look into this, but for some reason item.id was not working as expected, but this worked. I mentioned this below, but  the id and name kind of being similar in this case is confusing...

Reply:
definitely down to brainstorm ways to change the ID if it will make finding solutions easier!

Reply:
maybe if we rename an itemName to userInput it'll cause less confusion?
and the error Lindsey posted below caused by this line of code
 `const itemToUpdate = items.find((item) => itemId === item.itemName);`
it workde bexouse you had in your list only items without punctuation, but as soon as you add item with punctuation itemId != itemName => error

Reply:
I would definitely lean towards renaming something if it causes confusion among the team. Whatever name works for the team should be fine, whether `userInput` or something else. 

I would definitely change the itemName to id, though, before merge since it seems to be causing issues when special characters are used. @djtaylor8 let me know if you have issues getting id to work and I can try to help provide some assistance around that.